app/views/shared/_object_changes.html.haml

```diff
@@ -12,17 +12,40 @@
         - if can? :revert_attribute, version
           %th Action
     %tbody
-      - if version.event!= 'destroy'
+      - if version.event != 'destroy'
         - version.changeset.reject{ |_, values| values[0].blank? && values[1].blank? }.each do |attribute, values|
           %tr
             %td= attribute
-            - if version.event != 'create'
-              %td= values[0].blank? ?  '-' : values[0]
-            %td= values[1].blank? ?  '-' : values[1]
+
```

Comment:
Since we have another change to make anyway, let's add a comment for the following if. So that people can understand all the magic happening below. Sth like:

> If the attribute is an associated model, show the value of the record it corresponds to, not just the ID. 
> Eg. when the version is of an Event, if the attribute is event_type_id it shows
> Workshop (ID: 116) instead of just 116